packages/connect/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@hoprnet/hopr-connect",
-  "version": "1.93.1-next.6",
```

Comment:
Please remove the version changes. I changed the PR target to the staging branch.